src/CoreWCF.Primitives/src/CoreWCF/DXD.cs

```diff
@@ -0,0 +1,652 @@
+namespace CoreWCF
+{
+    using System.Xml;
+    using System.Collections.Generic;
+
+    // NOTE: This is a dynamic dictionary of XmlDictionaryStrings for the Binary Encoder to dynamically encode should
+    // the string not exist in the static cache.
+    // When adding or removing memebers please keep the capacity of the XmlDictionary field current.
+    static class DXD
+    {
```

Comment:
Explicit accessor needed. Should be internal.

Reply:
Also indentation is wrong on the internal. And need blank line before namespace. You also haven't added accessor's to all the fields. VS can do the entire file for you at once.